docs/index.md

```diff
@@ -33,5 +33,5 @@ There are two categories of docs: [Guides](./guides/) and [API docs](./api/). Gu
 * [Removing Players](./guides/removing-players.md) - Helpful for using VideoJS in single page apps.
 
 ## API Docs
-- The most relevant API doc is the [player API doc](./api/vjs.Player.md).
-- [Full list of API Docs](./api/)
+- The most relevant API doc is the [player API doc](./guides/api.md).
```

Comment:
This makes the title and the link not match. The most relevant API doc *is* the player api doc as well.